BuildResidentialHPXML/measure.rb

```diff
@@ -4941,17 +4981,26 @@ def self.set_pv_systems(hpxml, runner, args, weather)
         year_modules_manufactured = [args[:pv_system_year_modules_manufactured_1], args[:pv_system_year_modules_manufactured_2]][i].get
       end
 
+      is_shared_system = false
+      if [args[:pv_system_is_shared_1], args[:pv_system_is_shared_2]][i]
+        is_shared_system = [args[:pv_system_is_shared_1], args[:pv_system_is_shared_2]][i]
+        building_max_power_output = [args[:pv_system_building_max_power_output_1], args[:pv_system_building_max_power_output_2]][i]
+        number_of_bedrooms_served = args[:geometry_building_num_bedrooms].get
+      end
+
       hpxml.pv_systems.add(id: "PVSystem#{i + 1}",
                            location: [args[:pv_system_location_1], args[:pv_system_location_2]][i],
                            module_type: module_type,
                            tracking: [args[:pv_system_tracking_1], args[:pv_system_tracking_2]][i],
                            array_azimuth: [args[:pv_system_array_azimuth_1], args[:pv_system_array_azimuth_2]][i],
                            array_tilt: get_absolute_tilt([args[:pv_system_array_tilt_1], args[:pv_system_array_tilt_2]][i], hpxml.roofs[-1].pitch, weather),
-                           max_power_output: [args[:pv_system_max_power_output_1], args[:pv_system_max_power_output_2]][i],
+                           max_power_output: max_power_output,
                            inverter_efficiency: inverter_efficiency,
                            system_losses_fraction: system_losses_fraction,
                            year_modules_manufactured: year_modules_manufactured,
-                           is_shared_system: false)
+                           is_shared_system: is_shared_system,
+                           building_max_power_output: building_max_power_output,
```

Comment:
This is good. Note that I've decided to simplify this in a pending PR by removing `building_max_power_output`; you'll just set a single `max_power_output` for both shared/individual PV systems. Just a heads up.